### PR TITLE
[3819] Fixed text 'Applied coupon code: {couponCode}' being shifted up in the cart

### PR DESCRIPTION
**Pull request by S-Arturs**
**Related issue(s):**
* Fixes https://github.com/scandipwa/scandipwa/issues/3819

**Problem:**
* Text 'Applied coupon code: {couponCode}' is shifted up in the cart

**In this PR:**
* Centered the text vertically and removed form padding on top to comply with the design

### CODE REVIEW

**Comment by aleksandrakorolova** [comment]
On desktop, it looks good. But on mobile it looks ugly. Please, fix it
Screenshot 2021-12-02 at 14 55 18
Screenshot 2021-12-02 at 14 55 07

**Expected result:**
Screenshot 2021-12-02 at 14 54 57

**Review by AleksandrsKondratjevs — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -32,3 +32,8 @@ runtime-packages/csa
 # build specifc files
 lerna-debug.log
 .eslintcache
+*.json
+*.json
+*.json
+*.json
+packages/scandipwa/i18n/fr_FR.json
```
Comment (AleksandrsKondratjevs) [inline .gitignore:39]: What for ?

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (S-Arturs) [reply .gitignore]: An accident while figuring out how to deal with a merge conflict. My apologies for spam...

File: .gitignore [hunk 2]
```diff
@@ -1,34 +0,0 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
-# dependencies
-**/node_modules
-**/.pnp
-.pnp.js
-
-# testing
```
Comment (AleksandrsKondratjevs) [inline .gitignore:8]: Now you decided to delete it ? :d

**Review by AleksandrsKondratjevs — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (S-Arturs) [reply .gitignore:8]: Did it delete the entire .gitignore? I'm just speechless :|

**Review by AleksandrsKondratjevs — APPROVED** [review]

**Pull request merged**